type-definitions/PropagateTask.d.ts

```diff
@@ -0,0 +1,15 @@
+import { Sink, Task } from './types';
+
+export function propagateTask<T>(time: number, value: T, sink: Sink<T>): Task;
```

Comment:
propagateTask takes a run function as its first argument.  The run function takes 4 args.  [Here's the flow defs for it](https://github.com/mostjs/core/blob/master/src/index.js.flow#L254-L265).  Since one of the 4 run function args is the actual PropagateTask instance itself, and it has the `active` field, which is useful (and used in a few places already), you may end up needing to have a specific PropagateTask type (I had to do that in the flow defs).